charts/rancher-monitoring/v0.0.2/charts/prometheus/templates/prometheus.yaml

```diff
@@ -12,9 +12,33 @@ spec:
   securityContext:
 {{ toYaml .Values.securityContext | indent 4 }}
 {{- end }}
-{{- if eq .Values.level "cluster" }}
-  listenLocal: true
   containers:
+  - name: prometheus-proxy
+    command:
+    - /bin/sh
+    - -c 
+    - cp /nginx/run-sh.tmpl /var/run/nginx-start.sh; chmod +x /var/run/nginx-start.sh; /var/run/nginx-start.sh
```

Comment:
Why not mount `run-sh.tmpl` to `/var/run/nginx-start.sh` as file instead of `cp` and `run`?

Reply:
CRD `Promethues` doesn't allow us to change the file permission on `volumeMounts`, we need some steps to do this.